src/Spline.cpp

```diff
@@ -216,7 +231,7 @@ double SplineBase::getCurvatureMax()
     if (has_curvature_max)
         return curvature_max;
 
-    double const delPara = getUnitParameter() * geometric_resolution;
+    double const delPara = geometric_resolution;
```

Comment:
It was a constant factor before and still is one. Which is wrong... One could multiply it with 0.1 to make it finer grained or something. Still the result may be unexpected.